test/test_quantized.py

```diff
@@ -148,6 +148,27 @@ def test_qrelu6(self, X):
                 self.assertEqual(qY, qY_hat,
                                  message="{} relu failed".format(name))
 
+    """Tests the correctness of the quantized::relu op."""
+    @given(X=hu.tensor(shapes=hu.array_shapes(1, 5, 1, 5),
+                       qparams=hu.qparams()))
+    def test_qrelu_leaky(self, X):
+        X, (scale, zero_point, torch_type) = X
+        alpha = 0.1
```

Comment:
Should we test different values for this? Though i like the choice of `0.1` for its pathological floating point representation :)